roles/vdm/tasks/sasdeployment_custom_resource.yaml

```diff
@@ -199,21 +205,21 @@
       args:
         chdir: "{{ ORCHESTRATION_TOOLING_DIRECTORY }}"
       register: sasdeployment
-    - name: sasdeployment custom resource - Write SAS Viya uninstall manifest
+    - name: Sasdeployment custom resource - Write SAS Viya uninstall manifest
       copy:
         content: "{{ sasdeployment.stdout }}"
         dest: "{{ ORCHESTRATION_TOOLING_UNINSTALL_MANIFEST }}"
         mode: "0660"
+
+- name: Sasdeployment custom resource - Create SAS Viya uninstall manifest V4_DEPLOYMENT_OPERATOR_ENABLED - Ansible
   when:
     - V4_DEPLOYMENT_OPERATOR_ENABLED
-    - deployment_tooling == "docker"
+    - deployment_tooling == "ansible"
```

Comment:
Was the logic initially wrong here? Not sure why there is the change between docker and ansible

Reply:
@thpang, the github difference engine makes it look like a change between docker and ansible but when I look at the whole file, it was just a relocation of the when clause for both tasks with no change to the logic, each deployment_tooling comparison remained the same for both tasks.